src/rcc.rs

```diff
@@ -322,10 +352,36 @@ impl CFGR {
 
         // Enable PLL
         if let Some(pllmul_bits) = pllmul_bits {
-            self::inner::enable_pll(&mut self.rcc, &self.clock_src, pllmul_bits, ppre_bits, hpre_bits);
+            self::inner::enable_pll(
+                &mut self.rcc,
+                &self.clock_src,
+                pllmul_bits,
+                ppre_bits,
+                hpre_bits,
+            );
         } else {
             let sw_var = self::inner::get_sww(&self.clock_src);
 
+            // CRS is only available on devices with USB and HSI48
+            #[cfg(any(
+                feature = "stm32f042",
+                feature = "stm32f048",
+                feature = "stm32f072",
+                feature = "stm32f078",
+            ))]
+            match self.crs {
```

Comment:
`if let Some(crs) = self.crs {` ?

Reply:
Yields a nice error message that `#[cfg]` gating is not yet supported on `if`. 😅